code/modules/paperwork/paper_biscuit.dm

```diff
@@ -0,0 +1,80 @@
+/obj/item/folder/biscuit
+	name = "A biscuit card"
+	desc = "An biscuit card"
+	icon_state = "paperbiscuit"
+	bg_color = "#b88f3d"
+	var/cracked = FALSE //Is biscuit cracked open or not?
+
+/obj/item/folder/biscuit/suicide_act(mob/living/user)
+	user.visible_message(span_suicide("[user] tries to eat the 'biscuit'! [user.p_theyre()] trying to commit suicide!"))
+	return TOXLOSS
+
+/obj/item/folder/biscuit/update_overlays()
+	. = ..()
+	if(contents.len && cracked) //Shows overlay only when it has content and is cracked open
+		. += "paperbiscuit_paper"
+		. -= "folder_paper"
+
+/obj/item/folder/biscuit/examine()
+	. = ..()
+	if(length(contents))
+		. += span_notice("To reach contents you need to crack it open.")
+
+/obj/item/folder/biscuit/proc/crack_check(mob/user)
+    if (cracked)
+        return TRUE
+    balloon_alert(user, "need to crack it open first!")
+    return FALSE
+
+//All next is done so you can't reach contents, or put any new contents when its not cracked open
+/obj/item/folder/biscuit/remove_item(obj/item/Item, mob/user)
+    if (crack_check(user))
+        return ..()
+
+/obj/item/folder/biscuit/attack_hand(mob/user, list/modifiers)
+	if (LAZYACCESS(modifiers, RIGHT_CLICK))
+		if (crack_check(user))
+			return ..()
+	else ..()
```

Comment:
```suggestion
	if (LAZYACCESS(modifiers, RIGHT_CLICK) && crack_check(user)
			return ..()
	return
```

Reply:
@san7890 this **broke** it! Are you sure this is really necessary?